luminlife-events.php

```diff
@@ -159,8 +163,32 @@ function lumin_events_shortcode($attributes) {
   if (isset($link_to)) {
     $optionsStr .= "linkTo: '${link_to}',";
   }
+  if (isset($origin_url)) {
+    $optionsStr .= "originUrl: '${origin_url}',";
+  }
+  if (isset($show_calendar_view)) {
+    $optionsStr .= "showCalendarView: ${show_calendar_view},";
+
+    if ($show_calendar_view) {
+      wp_enqueue_style('luminlife-calendar_style',
+        esc_url_raw(CALENDAR_STYLE_URL), '', null);
+    }
+  }
+  if (isset($background_color)) {
+    $optionsStr .= "backgroundColor: '${background_color}',";
+  }
+
   $optionsStr .= "}";
 
+  if (is_page()) {
```

Comment:
Moved this code from the `register_scripts` function so that we can apply the styles in the required order to allow overriding of the Full Calendar styles where necessary